DEVELOPMENT.md

```diff
@@ -14,6 +47,45 @@ Repository contains repository hook definition to auto-generate chart documentat
 When you have the pre-commit hook installed and `commit` a file, it will run the pre-commit hook. If there is a change generated in the `README.md`, the hook will fail and notify you about the new change. You can review the changes and run the `commit` again, now it should pass.
 
 
+## Build and test the project
+
+To build and test the project locally use maven. Build process uses helm charts and values yaml files in `src/main/charts` and generates `/target` folder.  
+All items in `Chart.yaml` and `values.yaml` could be overridden by `test/charts/config/[product]/` files. 
+
+```shell script
+mvn clean install -Dshared.pvc.name=shared-home-claim -Deks.ingress.domain=kube.jira-dev.com -Dkubernetes.target.namespace=jira
+```
+
+## Install helm charts 
+
+To install the helm charts for test purposes `helm_install.sh` can be used. This script accepts the parameters to customise the 
+installation and does the following:
+
+* Check the bash version to ensure is 5 or higher
+* Ensure jq is available
+* Ensure the `shared-home` is available and bootstrap nfs
+* Create the postgres database
+* Create elasticsearch
+* Package the selected product's Helm chart
+* Package the functest Helm chart
+* Install the product's Helm chart
+* Install the functest Helm chart
+* Wait until the Ingress we just created starts serving up non-error responses
+* Run the chart's tests
+
+Helm parameters are produced in the build process and stored in `test/config/[product]/helm_parameters`. 
+To install and test the product on the existing Kubernetes cluster run the following script:
+
+```shell script
+./src/test/scripts/helm_install.sh target/config/[product]/helm_parameters
+```
+
+To uninstall the test product and cleanup the resources run `helm_uninstall.sh`:
+
+```shell script
+./src/test/scripts/helm_uninstall.sh target/config/[product]/helm_parameters
+```
+
```

Comment:
Im really not sure if we should be advertising this stuff. helm_install.sh is something we use only for CI, and isnt used for "development" of the charts per se. It's caused problems/confusion in the past and Ive stopped using it myself. To be honest if a turn key solution is what we're trying to provide here then we should wait for the official Terraform scripts.